packages/x-tailwindcss/src/x-tailwind-plugin/plugin.ts

```diff
@@ -42,6 +42,7 @@ export default plugin.withOptions(
         }
       );
       helpers.addUtilities(deepMerge({}, utilities(helpers), options?.utilities?.(helpers)));
+      helpers.addVariant('selected', '&.selected');
```

Comment:
Variant for the `x-selected` (`selected:`) class, so we can use it with Tailwind like other variants (`haver:`, `active:`, etc)